pkg/dashboard/api_test.go

```diff
@@ -64,3 +65,47 @@ func TestConfigureStatic(t *testing.T) {
 
 	assert.Equal(t, w.Code, http.StatusOK)
 }
+
+func TestConfigureRoutes(t *testing.T) {
+	w := httptest.NewRecorder()
+
+	req, err := http.NewRequest("GET", "/status", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Create a API Engine
+	api := gin.Default()
+
+	// Required arguements for route configuration
+	abortWeb := make(utils.ControlChan)
+	data := &objects.DataLayer{
+		StatusInfo: &objects.StatusInfo{},
+		Cache:      objects.NewCache(),
+	}
+
+	// Configure routes to API engine
+	configureRoutes(abortWeb, data, api)
+
+	// Start the server
+	api.ServeHTTP(w, req)
+
+	assert.Equal(t, w.Code, http.StatusOK)
+}
+
+func TestContextSetter(t *testing.T) {
+	w := httptest.NewRecorder()
+	con := GetTestGinContext(w)
+
+	// Required arguements
+	data, _ := objects.NewDataLayer("TestSpace", "T-1", objects.NewHelmConfig)
```

Comment:
The error here should be handled to fail the test, not ignored

Reply:
Ok sir.
Sir can I use like t.Fatal(err) ?

Reply:
Yes, that's what you should use